TicTacToe/UserSelection.fs

```diff
@@ -55,7 +56,17 @@ let askIfInverted (print, input : unit -> string) =
     let answer = input().ToCharArray()
     answer.[0] = '1'
 
-let askIfMinimax (print, input : unit -> string) = 
-    print ("If you would like to play against the minimax algorithm press the 1 key.  Press any other key for a rules algorithm.\n")
-    let answer = input().ToCharArray()
-    answer.[0] = '1'
\ No newline at end of file
+let rec askAlgorithmSeed (print, input : unit -> string, gameSize) = 
```

Comment:
I understand what "algorithm seed" after reading through the code where it's used but it's not exactly a clear name